src/renderer/vite.config.js

```diff
@@ -8,15 +8,15 @@ export default defineConfig({
   },
   define: {
     "process.env.VITE_APP_VERSION": JSON.stringify(
-      require("./package.json").version
+      require("../../package.json").version
```

Comment:
do we need to make a separate package.json or is referring to the root one fine?